configs/laramie_bmi_config_ftm.txt

```diff
@@ -4,13 +4,16 @@ end_time_d=1035.91
 dt_s=3600.0
```

Comment:
Same question here: Do we need to add units to this config file similar to the cfe config? [units]

Reply:
done